c10/util/qint8.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <cstdint>
+
+namespace c10 {
+
+/**
+ * This is the data type for quantized Tensors. Right now we only have
+ * qint8 which is for 8 bit Tensors, we might have 4 bit, 2 bit or 1 bit
+ * data types in the future.
+ */
+struct alignas(1) qint8 {
+  uint8_t val_;
```

Comment:
Does the `uint8_t` assume the quantization is asymmetric? For symmetric quantization, the value could be either `unsigned` or `signed`. How do we know? Will you put a flag in the symmetric quantizer for it?

Reply:
An alternative is to share the same class for symmetric and asymmetric quantizer and decide whether the quantization is symmetric via the zero point value, i.e. 0 for unsigned symmetric and 128 for signed symmetric.

Reply:
I think we can have new data types for symmetric quantization

Reply:
we can add these support later if there is a need, right now we only have per tensor affine quantization with unsigned int8 as data type.